### DISCUSSION TITLE
how to enable and use doge

### DISCUSSION THREAD

**Discussion by aviNaftalis** [LazyVim/LazyVim#3279] [Q&A] [answered]
I'm trying to use [doge](https://github.com/kkoomen/vim-doge?tab=readme-ov-file) - so far I updated my `lua/plugins/lazyvim.lua` and added

```
{
  "kkoomen/vim-doge",
}
```

restarted vim and got it installed.

However, I don't see how to config it correctly and use it.

Questions:

1. Is there an example config I can copy? If not, where should I config [this](https://github.com/kkoomen/vim-doge?tab=readme-ov-file)
2. Is there a way to install new plugins from the lazy menu, instead of adding it manually to the `lua/plugins/lazyvim.lua` like I did? Some plugins are installable from there, but I couldn't find doge...

**Comment by dpetka2001** [comment, marked as answer]
1. You should put the options you want as `vim.g.name_of_option` in the `init`function.
2. You have to write the the plugin spec into a file `/lua/plugins/` directory. There's no such thing as VsCode Marketplace in Neovim ecosystem yet.

**Reply by aviNaftalis** [reply]
not sure what you mean by 1. I have only `lua/plugins/options` / `lua/plugins/keymaps` / `lua/plugins/lazyvim` / `lua/plugins/SOME OTHER STUFF`

**Reply by dpetka2001** [reply]
Look at [lazy.nvim's examples](https://github.com/folke/lazy.nvim#examples), particularly how `vim-startuptime` (which is a vim plugin) is setup. You should do something similar.

**Reply by aviNaftalis** [reply]
Thanks. I was missing the `:call doge#install()` to make it work. Eventually it wasn't as good as I expected - Co-Pilot is better in terms of adding the correct tpyes (python) and adding some sensible description
